README.md

```diff
@@ -1,2 +1,87 @@
-# HelloMicroverse
-This is hello Microverse project ( Module 1 )
+📗 Table of Contents
+📖 About the Project
+🛠 Built With
+Tech Stack
+Key Features
+🚀 Live Demo
+💻 Getting Started
+Setup
+Prerequisites
+Install
+Usage
+Run tests
+Deployment
+👥 Authors
+🔭 Future Features
+🤝 Contributing
+⭐️ Show your support
+🙏 Acknowledgements
+❓ FAQ (OPTIONAL)
+📝 License
+📖 Hello Microverse
+Describe your project in 1 or 2 sentences.
+
+Hello Microverse is a project that we made as a part of module 1
+
+🛠 Built With
+Tech Stack
+
+Client
+HTML / CSS
+Key Features
+This is Hello Microverse project part of module 1
+
+(back to top)
+
+🚀 Live Demo
+Soon
+
+Live Demo Link
+(back to top)
+
+💻 Getting Started
+Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+Prerequisites
+In order to run this project you need:
+◦ Installation of Git
+◦ Installation of NPM
+◦ A working browser
+◦ Internet connection
+◦ A code editor like Visual Studio Code
+◦ Installation of Live Server plugging in Visual Studio Code
+
+Setup
+Clone this repository to your desired folder:
+follow the steps following the link on how to clone repo to a local machine : 
+https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository
+
+
+(back to top)
+
+👥 Authors
+
+👤 Author
+
+GitHub: @Aminocodex
+
+🤝 Contributing Contributions, issues, and feature requests are welcome!
+
+(back to top)
+
+⭐️ Show your support Write a message to encourage readers to support your project
+
+If you like this project, give it a ⭐️.
+
+(back to top)
+
+🙏 Acknowledgments Give credit to everyone who inspired your codebase.
+
+a big thanks for anyone whose code was used to make this project operational
+(back to top)
+
+📝 License This project is MIT licensed.
+
+NOTE: we recommend using the MIT license - you can set it up quickly by using templates available on GitHub. You can also use any other license if you wish.
```

Comment:
- [ ] Please include an MIT license file in your project. The link to your license should be included in your README file. The MIT License is a popular open-source license that allows others to use, modify, and distribute your code as long as they give credit to you the original author, and include the original license text. You can use this [link](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to learn how to add one to your project wink.
![mit2](https://user-images.githubusercontent.com/70483835/218863323-907820c7-4bd8-4a7a-9bc8-bf145b3026c5.png)